src/main/java/com/spotinst/sdkjava/example/OceanGKELaunchSpecUsageExample.java

```diff
@@ -0,0 +1,184 @@
+package com.spotinst.sdkjava.example;
+
+import com.spotinst.sdkjava.SpotinstClient;
+import com.spotinst.sdkjava.model.*;
+import com.spotinst.sdkjava.model.bl.gcp.*;
+
+import java.io.IOException;
+import java.util.*;
+
+public class OceanGKELaunchSpecUsageExample {
```

Comment:
To be reviewed and tested (by running sample) when all other comments are fixed.